controllers/questions.ts

```diff
@@ -0,0 +1,63 @@
+const crypto = require("crypto");
+import { Request, Response } from "express";
+
+import { Question } from "../types/questions";
+import { CustomRequest, CustomResponse } from "../types/global";
+
+const logger = require("../utils/logger.ts");
+const { HEADERS_FOR_SSE } = require("../constants/constants");
+const { INTERNAL_SERVER_ERROR } = require("../constants/errorMessages");
+
+const questionQuery = require("../models/questions");
+
+let clients = [];
+
+function sendQuestionToAll(newQuestion: Question, res: CustomResponse) {
+  clients.forEach((client) => client.res.write(`data: ${JSON.stringify(newQuestion)}\n\n`));
+
+  res.status(201).send({
+    message: "Question created and sent successfully to connected peers",
+    data: newQuestion,
+  });
+}
+
+const createQuestion = async (req: CustomRequest, res: CustomResponse) => {
+  try {
+    const questionId = crypto.randomUUID({ disableEntropyCache: true });
+    const question = await questionQuery.createQuestion({ ...req.body, id: questionId });
+    return sendQuestionToAll(question, res);
+  } catch (error) {
+    logger.error(`Error while creating question: ${error}`);
+    return res.boom.badImplementation(INTERNAL_SERVER_ERROR);
+  }
+};
+
+// eslint-disable-next-line consistent-return
```

Comment:
because this are different kind of APIs. In this the connection doesn't closes instead it stays until the client doesn't disconnect explicitly or session gets closed. So for normal API we return the response and close the connection but in this case a return will not be there and a stream of communication will be there to write data everytime new data comes.

Hope this clears the doubt